arlo-client/wdio.conf.js

```diff
@@ -90,7 +90,7 @@ exports.config = {
   //
   // If you only want to run your tests until a specific amount of tests have failed use
   // bail (default is 0 - don't bail, run all tests).
-  bail: 0,
+  bail: 1,
```

Comment:
@jonahkagan Just want to check if you had a reason for this being here other than time-efficiency in debugging. I would recommend not leaving it in there if so, since when there is a real failure, sometimes it happens on multiple tests and it helps to triangulate what's going on.

Reply:
My thought was that we didn't catch this issue before because there were way too many unrelated log lines to look through and isolate the actual source of the failure. I thought we'd be more likely to catch failures in the future if there were less extra log lines.

Your reasoning is compelling too. I don't have a strong opinion. If you'd rather I change it back I can